### DISCUSSION TITLE
Adding new language translation

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1722] [Q&A]
Where to add a new language translation such as ar.po not included in the parent theme (understrap)?

**Comment by bacoords** [comment]
You can set up the child theme text domain: https://developer.wordpress.org/reference/functions/load_child_theme_textdomain/ and insert them in a `/languages/` folder

**Reply by Fedora-Man** [reply]
I did this but still not working! It worked when I put the translation files in the parent theme (understrap) but it does not work in the child theme (understrap-child)?

This is the function.php:
```
function add_child_theme_textdomain() {
	load_child_theme_textdomain( 'understrap-child', get_template_directory() . '/languages' );
}
add_action( 'after_setup_theme', 'add_child_theme_textdomain' );
```

**Comment by 0dp** [comment]
get_template_directory should probably be get_stylesheet_directory

> In the case a child theme is being used, the absolute path to the parent theme directory will be returned. Use get_stylesheet_directory() to get the absolute path to the child theme directory.

https://developer.wordpress.org/reference/functions/get_template_directory/

**Reply by Fedora-Man** [reply]
I tried both but still not working but it worked when I used the parent theme (understrap) as the textdomain!
